context/themeContext.tsx

```diff
@@ -2,31 +2,35 @@
 import { createContext, useContext, useState, useEffect } from "react";
 
 interface ThemeContextType {
-  darkModeTheme: boolean;
+  theme: string;
   toggleTheme: () => void;
 }
 
 const ThemeContext = createContext<ThemeContextType>({
-  darkModeTheme: true,
+  theme: "dark",
   toggleTheme: () => {},
 });
 
 export function ThemeProvider({ children }) {
-  const [darkModeTheme, setDarkModeTheme] = useState(false);
+  const [theme, setTheme] = useState("dark");

```

Comment:
ustawiłbym gdzieś typescriptowy enum

enum AppModeTypes = {
   DARK = 'dark',
   LIGHT = 'light',
}

i używał zamiast wartości string, wartości enum. to samo w powyższym interfejsie